packages/remix/src/ssr/authenticateRequest.ts

```diff
@@ -75,6 +75,21 @@ export function authenticateRequest(args: LoaderFunctionArgs, opts: RootAuthLoad
   const signInUrl =
     opts.signInUrl || getEnvVariable('CLERK_SIGN_IN_URL') || (context?.CLERK_SIGN_IN_URL as string) || '';
 
+  const signUpUrl =
+    opts.signUpUrl || getEnvVariable('CLERK_SIGN_UP_URL') || (context?.CLERK_SIGN_UP_URL as string) || '';
+
+  const afterSignInUrl =
+    opts.afterSignInUrl ||
+    getEnvVariable('CLERK_AFTER_SIGN_IN_URL') ||
+    (context?.CLERK_AFTER_SIGN_IN_URL as string) ||
+    '';
+
+  const afterSignUpUrl =
+    opts.afterSignUpUrl ||
+    getEnvVariable('CLERK_AFTER_SIGN_UP_URL') ||
+    (context?.CLERK_AFTER_SIGN_UP_URL as string) ||
+    '';
```

Comment:
☁️ If we had a centralized config object, probably this (and the 3 following added lines) would have been the change to support the Repo level config for remix. Right?

Reply:
We'd also have to include the options in `authenticateRequest`.

Reply:
@dimkl That is correct. We need to bring this up again now that we have 2 packages using repo-level config otherwise, this will become unwieldy